linker/shared/src/main/scala/org/scalajs/linker/interface/unstable/OutputFileImpl.scala

```diff
@@ -31,7 +30,17 @@ abstract class OutputFileImpl extends LinkerOutput.File {
         else Future.successful(())
       }
 
-      writeLoop().finallyWith(chan.close())
+      writeLoop()
+      // finallyWith(chan.close())
```

Comment:
Consider extracting that in a
```scala
private def finallyWith[A](f: Future[A], finallyFuture: => Future[Unit]): Future[A]
```
instead of relying on a comment.